opendistro-elasticsearch-alerting.release-notes.md

```diff
@@ -1,4 +1,10 @@
-## Version 1.4.0.0 (Current)
+## Version 1.6.0.0 (Current)
+
+### New Features
+  * Adds support for Elasticsearch 7.6.1 - [PR #186](https://github.com/opendistro-for-elasticsearch/alerting/pull/186)
+  * Adds support for Elasticsearch 7.5.2 - [PR #185](https://github.com/opendistro-for-elasticsearch/alerting/pull/185)
```

Comment:
Let's remove this one, doesn't make sense to include as a new feature in Version 1.6.0.0 as they can't actually use this for 7.5.2 in this version because it's been overwritten by the 7.6.1 support.

Reply:
This make sense :ok_hand: